model/lifecycle.go

```diff
@@ -391,28 +391,17 @@ func RestartVersion(versionId string, taskIds []string, abortInProgress bool, ca
 	if err != nil && !adb.ResultsNotFound(err) {
 		return errors.WithStack(err)
 	}
+	for i := len(finishedTasks) - 1; i >= 0; i-- {
```

Comment:
why are you  doing this in this order?

Reply:
i get this may be temporary but it could use a comment explaining why we drop execution tasks